pkg/scheduler/apis/config/validation/validation_pluginargs_test.go

```diff
@@ -513,7 +514,7 @@ func assertErr(t *testing.T, wantErr string, gotErr error) {
 		if gotErr == nil {
 			t.Fatalf("\nwant err to be:\n\t%s\ngot:\n\tnil", wantErr)
 		}
-		if gotErr.Error() != wantErr {
+		if !errors.Is(gotErr, errors.New(wantErr)) {
```

Comment:
while I like this, it's unrelated. Could you do it in a separate PR, you can reference #95047